### ISSUE TITLE
Send PRs from branches in main repo or personal fork?

### ISSUE THREAD

**Issue by strugee** [openopensource/openopensource.github.io#15]
Forked from a discussion in #10; go read that if you want the full background. tl;dr: does it clutter the upstream repository if all collaborators put their development branches there? Or should contributors use their personal fork?

As I stated in #10 I'm personally of the opinion that this doesn't really matter, and branch cluttering generally isn't really a big deal (even if it _does_ appear in the dropdown on the repo front page which was posted in a screenshot in the other issue).

Note also that when authoring a change using GitHub's web interface, it'll default to creating a branch in the upstream repo, not your fork.

**Comment by eush77** [comment]
Maybe prefix the branch name with the collaborator name? You could have a branch `strugee/navbar-fixes-rename` and I can have `eush77/colors`, all in one repo, and it would be perfectly clear whose branches they are.

**Comment by strugee** [comment]
Yeah, that seems like a good approach. I'm starting to feel like this is maybe something that might vary too much per-project for us to say anything reasonable here. For smaller projects, for example, that seems like unnecessary overhead. Though maybe that's just me thinking about how often I'd forget ;)
